app/javascript/vue/tasks/nomenclature/by_source/components/tables/taxon_names_row.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <tr>
+    <td>
+      <input
+        class="pages"
+        type="text"
+        placeholder="Pages"
+        @input="changePage"
+        v-model="citation.pages">
+    </td>
+    <td>
+      <span>
+        <a
+          v-html="citation.citation_object.object_tag"
+          @click="showObject()"/> 
+        {{ legend }}
+      </span>
+    </td>
+    <td>
+      <radial-annotator :global-id="citation.citation_object.global_id"/>
+    </td>
+    <td>
+      <otu-radial
+        :taxon-id="citation.citation_object_id"
+        :redirect="false"/>
+    </td>
+    <td>
+      <span
+        class="button circle-button btn-delete"
+        @click="removeMe()"/>
+    </td>
+  </tr>
+</template>
+
+<script>
+
+  import RadialAnnotator from '../../../../../components/annotator/annotator'
```

Comment:
@jlpereira @jrflood This type of relative annotation makes me super nervous. What are the alternatives?  Can we at least go from `/` forward?

Reply:
Done.

Now this could be imported like:

`import RadialAnnotator from 'components/annotator/annotator'`

Remember this change was made in development, needs to be merge here to change that line

Reply:
*MUCH* better, thanks!